content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,241 @@
+---
+reviewers:
+- jsafrane
+title: Create static Pods
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+
+*Static Pods* are managed directly by kubelet daemon on a specific node,
+without the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
+observing it.
+Unlike Pods that are managed by the control plane (eg, a
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}};
+instead, kubelet itself watches it and restarts it when it crashes.
+There are no health checks for the containers in a static pod.
+
+Static pods are always bound to one {{< glossary_tooltip term_id="kubelet" >}} daemon on a specific node.
```

Comment:
General comment:
Our style is to capitalize, or camel case, the names of Kubernetes objects. So change pod to Pod throughout the topic.

Reply:
Changed!